docs/docs/flags/output.1.md

```diff
@@ -48,7 +48,10 @@ Other options:
   - **exec-env**: When tracing execve/execveat, show the environment variables that were used for execution.
   - **relative-time**: Use relative timestamp instead of wall timestamp for events.
   - **exec-hash**: When tracing some file related events, show the file hash (sha256).
-    - Affected events: sched_process_exec, shared_object_loaded
+    - Affected events: *sched_process_exec*, *shared_object_loaded*
+    - **pathname** option recalculates the file hash if the inode's creation time (ctime) differs, which can occur in different namespaces even for identical pathnames.
```

Comment:
This documentation is not updated to the new options

Reply:
Thanks for catching.
Unfortunately `man` pages aren't working for me for whatever reason so it's easier to forget.